test/hermez-account.test.js

```diff
@@ -53,4 +55,30 @@ describe("Hermez account", () => {
         const res = txUtils.verifyTxSig(tx);
         expect(res).to.be.equal(true);
     });
+    it("Sign withdraw Bjj", async () => {
+        const account = new Account(1);
+
+        // bjj signature
+        const zkInputs = {
+            rootState: Scalar.sub(Scalar.shl(1, 256), 1),
+            ethAddrCaller: Scalar.sub(Scalar.shl(1, 160), 1),
+            ethAddrBeneficiary: Scalar.sub(Scalar.shl(1, 160), 1),
+            tokenID: Scalar.sub(Scalar.shl(1, 32), 1),
+            exitBalance: Scalar.sub(Scalar.shl(1, 192), 1),
+            idx: Scalar.sub(Scalar.shl(1, 48), 1),
+        };
+
+        const signature = account.signWithdrawBjj(zkInputs);
+
+        expect(signature.R8[0]).to.be.not.equal(undefined);
+        expect(signature.R8[1]).to.be.not.equal(undefined);
+        expect(signature.S).to.be.not.equal(undefined);
+
+        // Verify transaction
+        const h = withdrawUtils.hashInputsWithdrawBjj(zkInputs);
```

Comment:
Maybe it is worth to add a function in `withdraw-utils.test.js` such as `verifyWithdrawBjjSig`, quite similar to https://github.com/hermeznetwork/commonjs/blob/master/src/tx-utils.js#L203